src/components/HomePage.jsx

```diff
@@ -1,45 +1,38 @@
-import React from "react";
+import React from 'react';
 import { useEffect, useState } from 'react';
 import { fetchArticles } from '../ApiRequests';
 import './HomePage.css';
 import { Link } from "react-router-dom";
 
-
-function Homepage() {
+function HomePage() {
   const [articles, setArticles] = useState([]);
-  const [isLoading, setIsLoading] = useState(true)
-    useEffect(() => {
-      fetchArticles().then((articlesFromApi) => {
-      setArticles(articlesFromApi)
-       setIsLoading(false)
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    fetchArticles().then((articlesFromApi) => {
+      setArticles(articlesFromApi);
+      setIsLoading(false);
     });
-    }, []);
+  }, []);
 
-    if (isLoading) {
-    return <div>Is Loading....</div>;
+  if (isLoading) {
+    return <div> Loading....</div>;
   }
   return (
-   
     <div className="homepage_wrapper">
-      {
-        articles.map((article, index) => (
-        <div key={index} className="article_card" >
+      {articles.map((article, index) => (
+        <div key={index} className="article_card">
```

Comment:
When listing articles you should use the semantic list tags, `ul` / `ol` and `li`'s instead of `div`'s which have no meaning. Swap these out.